El Gordo Franses volverá a amar este PR.

SagaStars opened the pull request:
12 Comidas nuevas de 4 frutas nuevas, este PR es parte de la bounty hispania: https://spacestation13.es/Foro/showthread.php?tid=785&pid=2007#pid2007
![YES](https://user-images.githubusercontent.com/55407528/80851120-a893f400-8bed-11ea-88fa-abe280021e2b.png)

## Recetas de bebidas (Mezclando sin herramientas)

**Super-Healthy Prickly Pear Juice**
5u de azucar, 5u de cactusjuice, 5u de lemonjuice y 1u de ice
**Refined Nispero Juice**
10u de azucar, 1u de lemonjuice, 1u de nisperojuice y 1u de ice
**Anonna Cream**
5u de azucar, 5u de water, 1u de anonnajuice, 2u de cornoil
**Anonna and Blueberries**
1u de berryjuice y 1u de anonnajuice

----
## Recetas de Horno

**Cheese Anonna Cake**
3 doughs, 1 cheesewedge, 1 anonna. 5u de milk y 15u de azucar.
**Pancake de Mermelada**
1 flatdough, 1 nispero, 1 berries. 5u de azucar y 1u de sodiumchloride

----
## Recetas de Microondas

**Ensalada de Cactus**
1 cabbage, 1 tomato, 1 pricklypear. 1u de sodiumchloride y 1u de blackpepper

----
### Recetas de Candy

**Mousse**
1 avocado. 5u de azucar, 5u de leche y 5u de cocoa
**Nispero Candy**
1 nispero, 1 lemon. 10u de azucar y 5u de cornoil

----
### Recetas de Grill

**Filete con Cactus**
1 meat/synthmeat, 1 pricklypear. 1u de sodiumchloride y 1u de blackpepper

----
### Recetas de Tablecraft(Sandwichs)

**Avocado Sandwich**
2 avocadoslice, 2 toast y 1 mayonnaise
**Bacon Rolls**
1 avocadoslice y 1 bacon


## Changelog
:cl:
add: Sandwich of Avocado
add: Salad of Pickly Pear
add: Mousse
add: Cheese Anonna Cake
add: Smoothie of Prickly Pear
add: Glass of Anonna Cream
add: Bacon Rolls
add: Filetes con Cactus
add: Cóctel of Anonna and Blueberries
add: Nispero Candy
add: Glass of Refined Nispero Juice
add: Pancake de Mermelada
add: Prickly Pear Cactus Juice
imageadd: Nuevas comidas, Nuevas sprites
fix: Un par de typos de bebidas de hispania
/:cl:

SagaStars committed:
COMEBACK

Silverexperience reviewed (changes requested):
Pon en el changelog los imageadd de los sprite.

SagaStars committed:
juicer stuff

SagaStars committed:
No vieron nada-

SagaStars committed:
No vieron nada 2-

Silverexperience reviewed (changes requested):
Las diferentes frutas que añades y que dan un un juice, deberian funcionar con juice
El mousses no sale si completas la receta.
Deberias dar a entender mejor que es en Nispero Candy y el palstic-cup
Tienes unos runtimes por la falta de definiciones en los ingredientes.

Evankhell561 commented:
> Las diferentes frutas que añades y que daun un juice, deberian funcionar con juice
> El mousses no sale si completas la receta.
> Deberias dar a entender mejor que es en Nispero Candy y el palstic-cup
> Tienes unos runtimes por la falta de definiciones en los ingredientes.

Lo que yo decia, estas comidas fueron hechas por un chef amateur.

SagaStars committed:
WIP (Lee desc)

+ Ahora las frutas  se les extrae los reagents apartir de "juice"
+ Cambios a las sprites de las cups y el nisperocandy
+ Se le sube un poco el bitesize al nisperocandy para que no sea de 30 mordidas para comer.
+ Se arreglan unos 8 runtimes
- (WIP) Arreglar otros 4 runtimes

SagaStars commented:
Asi se ven ahora:
![image](https://user-images.githubusercontent.com/55407528/80923355-363c2480-8d51-11ea-82b3-cb2df4335cd8.png)

SagaStars committed:
Se me olvido este pedazin en el otro commit :p

SagaStars committed:
VICTORY!

+ Chu chu runtimes
+ La ensalada se come mas rapido, se me olvido que le habia bajado el bitesize :hehe:

Silverexperience reviewed (changes requested):
Podrias poner comentarios en lo que añades en paradise, para que se sepa lo que es nuestro y lo que no.
Saltan estos warnings.
![imagen](https://user-images.githubusercontent.com/58746682/80928081-74adff80-8da2-11ea-8c4b-a4059e464106.png)

SagaStars committed:
arreglao
